composer.json

```diff
@@ -15,17 +15,6 @@
             "email": "remy.abikhalil@canaltp.fr"
         }
     ],
-    "repositories": [
```

Comment:
Because TT dependencies are all open source and available in public packagist.org.
=> No need for declaring them anymore.